### ISSUE TITLE
Searching for songs on mobile is incredibly annoying

### ISSUE THREAD

**Issue by danieladugyan** [Dsek-LTH/web#553]
### Description
The songbook in the app has a search field that loses focus whenever the user stops typing.

### Steps to reproduce
1. Go to https://www.dsek.se/songbook on a phone.
2. Type some random input and then stop typing.
3. Notice that as soon as you stop typing, a query will be executed and this makes the keyboard disappear.

**Comment by fgren** [comment]
Potential solution: Increase the timeout between when the user stops typing and when the search gets made?

**Comment by fgren** [comment]
Partially fixed by #581 , but should probably be redone in a better way.

**Comment by Fiery-132** [comment]
There is an option `noScroll` in the [goto](https://svelte.dev/docs/kit/$app-navigation#goto) function. Maybe that could fix it?
```ts
await goto(`${i18n.route($page.url.pathname)}?${urlParams.toString()}`, {
  replaceState: true,
  noScroll: true,
});
```
